app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.styles.tsx

```diff
@@ -174,6 +174,10 @@ export const connectedStateCellIcon = css`
 
 export const connectedStateCellEvent = css`
 	margin-left: ${token('space.100')};
+    max-width: 400px;
```

Comment:
400 felt the most balanced when resizing and using massive amounts of env names... but if there was a standard happy to change!